streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBStore.java

```diff
@@ -118,9 +120,9 @@ void openDB(final ProcessorContext context) {
         userSpecifiedOptions = new RocksDBGenericOptionsToDbOptionsColumnFamilyOptionsAdapter(dbOptions, columnFamilyOptions);
 
         final BlockBasedTableConfig tableConfig = new BlockBasedTableConfig();
-        tableConfig.setBlockCacheSize(BLOCK_CACHE_SIZE);
+        tableConfig.setBlockCache(new LRUCache(BLOCK_CACHE_SIZE));
```

Comment:
Ditto for LRUCache: `Cache extends RocksObject`. We need to close it explicitly.

Reply:
Ack